dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/EnterpriseApplication.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *******************************************************************************/
+package com.ibm.websphere.simplicity.config;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+/**
+ * Represents a &lt;webApplication> configuration element
+ */
+public class EnterpriseApplication extends Application {
+
+    private String contextRoot;
+    private String annotationScanLibrary;
+
+    /**
+     * @return the contextRoot
+     */
+    public String getContextRoot() {
+        return contextRoot;
+    }
+
+    /**
+     * @param contextRoot the contextRoot to set
+     */
+    @XmlAttribute
+    public void setContextRoot(String contextRoot) {
+        this.contextRoot = contextRoot;
+    }
```

Comment:
```suggestion
    private String annotationScanLibrary;
```
contextRoot is not applicable to EnterpriseApplication from what I am seeing.  I believe you probably should add other attributes that are valid though like `defaultClientModule` and `webModuleClassPathLoader`.

Reply:
I was going for a minimum viable product, but sure, why not do the maximal impl. Its not that much larger. 

But are you sure about context root? In I see com.ibm.ws.app.manager.war/resources/OSGI-INF/metatype/metatype.xml

```    <OCD description="%earapp.desc" name="%earapp.name" id="com.ibm.ws.app.manager.earappcfg" ibm:alias="enterpriseApplication" ibm:extends="com.ibm.ws.app.manager">
        <AD id="type" name="internal" description="internal" type="String" default="ear" ibm:final="true"/>
        <AD id="context-root" name="internal" description="internal" ibm:final="true" required="false" type="String"/>```

Reply:
And context-root is internal.  It is not an external attribute to be set in server.xml.

Reply:
Got you, in that case I'll remove it.